src/test/app/Discrepancy_test.cpp

```diff
@@ -24,122 +24,124 @@
 #include <ripple/beast/core/LexicalCast.h>
 #include <ripple/protocol/JsonFields.h>
 #include <ripple/protocol/SField.h>
+#include <ripple/ledger/tests/PathSet.h>
```

Comment:
good point - the last four can definitely be removed. I am referencing types or defines from the remaining headers (field names, etc.), so I prefer to leave them instead of relying on a transitive include.